### ISSUE TITLE
Author on stories in story list is incorrect

### ISSUE THREAD

**Issue by thequbit** [hhroc/yellr-android#57]
The display name on the stories list for each story is incorrect.  It is currently just set to:

```
textViewPublishAuthor.setText(getString(R.string.fa_user) + " Tim Duffy/HHROC");
```

This needs to be pulled from the response from get_stories.json

**Comment by reustonium** [comment]
Fixed, will include in next PR

**Comment by reustonium** [comment]
closed via PR #62
